src/components/MediaSettings/MediaSettings.vue

```diff
@@ -507,12 +499,15 @@ export default {
 			if (page === 'video-verification') {
 				this.isPublicShareAuthSidebar = true
 			}
+
+			if (!BrowserStorage.getItem('audioInputDevicePreferred') || !BrowserStorage.getItem('videoInputDevicePreferred')) {
+				this.tabContent = 'devices'
+			}
 		},
 
 		closeModal() {
 			this.modal = false
 			this.updatedBackground = undefined
-			this.deviceIdChanged = false
```

Comment:
Maybe we need `this.tabContent = 'none'` in this place now, to dismiss previous state?

Reply:
There is no issue in maintaining the tab at which modal was closed.

Reply:
Just to note, it will be opened afterwards everytime you show modal again (until page reload)